src/Parser/LASParser.js

```diff
@@ -40,7 +60,11 @@ export default {
         if (options.out?.skip) {
             console.warn("Warning: options 'skip' not supported anymore");
         }
-        return lasLoader.parseFile(data, {
+
+        const useWorker = options.in.worker ?? true;
+        const loader = useWorker ? lasWorkerLoader : lasLoader;
+
+        return loader.parseFile(data, {
```

Comment:
Since data is an `ArrayBuffer` we should use `Transfer(data)` which [according to the doc](https://threads.js.org/usage-advanced#transferable-objects) speeds up perfs even more.

Reply:
Damn, forgot to push it!
I'll do it too when sending back a response in `LasParser.js`!